Loop/Info.plist

```diff
@@ -55,6 +55,8 @@
 	<true/>
 	<key>LoopLocalCacheDurationDays</key>
 	<string>$(LOOP_LOCAL_CACHE_DURATION_DAYS)</string>
+	<key>NFCReaderUsageDescription</key>
+	<string>The app uses NFC to pair with diabetes devices.</string>
```

Comment:
Yeah, this is a good point; Tidepool Loop doesn't strictly require NFC yet. I think it's fairly likely it will at some point in the future other cgm/pumps may use NFC for pairing, as it's a nice way to make things secure and identify devices without having the user have to type in codes. I don't have an easy way to turn this off just for Tidepool; we'd need to either make it part of the prepare build script, or add some other script to selectively enable/disable it. I was thinking we'd just take this as is now, as it doesn't hurt any existing processes/testing, and we can work on a solution when/if we need it.